### PR TITLE
docs: update security.md to match current migration-skill semantics

### PR DESCRIPTION
**Pull request by mvo5**
Trivial branch that updates the docs/security.md to reflect the current status of the migration-skill and to give a practical example.

### CODE REVIEW

File: docs/security.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Security policy
 
+Most of the security aspects of the system will be done via skills and
+slots. However for compatibility with the 15.04 snappy architecture
```
Comment (zyga) [inline docs/security.md]: s/slots/skill slots/

File: docs/security.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+there is a special skill called `migration-skill` that can be used to
```
Comment (zyga) [inline docs/security.md]: s/special skill/special skill type/

**Comment by zyga** [comment]
Please tweak the two words indicated, then LGTM

**Comment by mvo5** [comment]
Thanks @zyga ! I fixed the wording now.

**Pull request merged**